### ISSUE TITLE
Report formatted files

### ISSUE THREAD

**Issue by kindkaktus** [millermedeiros/esformatter#385]
It will be very handy to have CLI/configuration setting to report file names which have been altered when formatted in-place (-i option).

**Comment by millermedeiros** [comment]
yeah, we should probably add a new flag that toggles the behavior - it should not list the file names by default.

**Comment by millermedeiros** [comment]
maybe just a `--verbose` option on the cli. see also #388
